solver/src/settlement_submission/flashbots_api.rs

```diff
@@ -51,4 +70,23 @@ impl FlashbotsApi {
         ensure!(status.is_success(), "status {}: {:?}", status, body);
         Ok(())
     }
+
+    /// Query status of a previously submitted transaction.
+    pub async fn status(&self, bundle_id: &str) -> Result<()> {
```

Comment:
is this method used?

Reply:
Not currently, however I plan to use it in the following PR related to transaction strategies refactoring.